src/components/Search.js

```diff
@@ -196,6 +200,9 @@ Search.propTypes = {
   autoTrigger: PropTypes.bool,
   triggerDelay: PropTypes.number,
   onSelect: PropTypes.func.isRequired,
+  popupProps: PropTypes.shape({
```

Comment:
Why is it optional here, but `isRequired` in `DatePicker`. Pls be consistent

Reply:
Correct me if I'm wrong. I have not made popupProps as required in DatePicker.js 

### Search.js
![Screenshot 2020-07-30 at 11 01 53 AM](https://user-images.githubusercontent.com/52645649/88875597-1c2cb800-d254-11ea-9918-f16bd008b243.png)
### DatePicker.js
![Screenshot 2020-07-30 at 10 47 38 AM](https://user-images.githubusercontent.com/52645649/88875664-3cf50d80-d254-11ea-9aeb-b13be5ecf9e6.png)

Reply:
Sorry, my bad, I misread